templates/types/streaming/nextjs/app/components/ui/chat/widgets/InterpreterCard.tsx

```diff
@@ -0,0 +1,34 @@
+import { getStaticFileDataUrl } from "../../lib/url";
+
+export interface InterpreterData {
+  isError: boolean;
+  extraResult: Array<{
+    type: string;
+    url: string;
+    filename: string;
+  }>;
+  logs: {
+    stderr: string[];
+    stdout: string[];
+  };
+}
+
+export function InterpreterCard({ data }: { data: InterpreterData }) {
+  const { isError, extraResult } = data;
+  if (isError || !extraResult.length) return null;
+  return (
+    <div className="space-x-2">
+      <span className="font-semibold">Output Files:</span>
```

Comment:
![image](https://github.com/run-llama/create-llama/assets/17126/c9a9c918-6953-49bb-bdb9-05fd54a71377)
our tool output widget is just linking the output files?

Reply:
yes probably a next step (see above), I'll make a note for a "inline widget ticket"